CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 0.44.0 / 2020-12-02
+
+* [CHANGE] Fix child routes support in AlertmanagerConfig. #3703
+* [FEATURE] Add Slack receiver type to AlertmanagerConfig CRD. #3618
+* [FEATURE] Add WeChat receiver type to AlertmanagerConfig CRD. #3619
+* [FEATURE] Add Email receiver type to AlertmanagerConfig CRD. #3692
+* [FEATURE] Add Pushover receiver type to AlertmanagerConfig CRD. #3697
+* [FEATURE] Add VictorOps receiver type to AlertmanagerConfig CRD. #3701
+* [FEATURE] Add option to allow configuring object storage for Thanos. #3668
+* [FEATURE] Add TLS support for remote read. #3714
+* [ENHANCEMENT] Include EnforcedSampleLimit as a metric. #3617
+* [ENHANCEMENT] Adjust config reloader memory requests and limits. #3660
+* [ENHANCEMENT] Add settings for alertmanager clusters #3663
+* [ENHANCEMENT] Handle all possible reciever types in AlertmanagerConfig. #3689
```

Comment:
it's a `[BUGFIX]` for https://github.com/prometheus-operator/prometheus-operator/issues/3665

Reply:
Good catch! :+1: